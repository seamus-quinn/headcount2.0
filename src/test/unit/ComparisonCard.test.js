// import React from 'react';
// import { shallow, mount } from 'enzyme';
// import ComparisonCard from  '../../ComparisonCard';

// describe('ComparisonCard', () => {
//   let mockComparisonInfo = {compared: 2}

//   let mockSchoolRepository = {
//     findAverage: jest.fn(),
//     compareDistrictAverages: jest.fn()
//   };

//   let comparisonCard = mount(<ComparisonCard schoolRepository={mockSchoolRepository} comparedSchools={mockSchoolRepository}/>);

  
// });
