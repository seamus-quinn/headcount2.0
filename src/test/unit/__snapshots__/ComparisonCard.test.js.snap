// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComparisonCard should match snapshot 1`] = `undefined`;
